.github/workflows/dev.yaml

```diff
@@ -1,37 +1,38 @@
 name: "Integration Tests"
 on:
+  pull_request: {}
   pull_request_target:
     types:
       - labeled
 jobs:
   "build_image":
     runs-on: ubuntu-latest
-    if: ${{ github.event.label.name == 'ok to test' }}
+#    if: ${{ github.event.label.name == 'ok to test' }}
```

Comment:
this change also needs to get dropped when we merge (just making the note so we don't forget)